### PR TITLE
Demo churn wine script comments

### PR DESCRIPTION
**Pull request by AdityaJhaTrueFoundry**

### CODE REVIEW

**Comment by AdityaJhaTrueFoundry** [comment]
This PR by mistake has the changes of #32 too.
Fixing it, till then end-to-end-examples/diabetes-regression/batch-inference/* files can be ignored for comments

File: end-to-end-examples/churn-prediction/demo/demo.py [hunk 1]
```diff
@@ -4,8 +4,10 @@
 import requests
 from urllib.parse import urljoin
 
+# get the model deployment url from the environment variables
```
Comment (nikp1172) [inline end-to-end-examples/churn-prediction/demo/demo.py:7]:
```suggestion
# Endpoint of the deployed model
```

File: end-to-end-examples/churn-prediction/demo/demo.py [hunk 2]
```diff
@@ -17,12 +19,14 @@
     gr.Number(label="EstimatedSalary", value=101348.88)
 ]
 
-
+# prediction function
```
Comment (nikp1172) [inline end-to-end-examples/churn-prediction/demo/demo.py:22]: remove

File: end-to-end-examples/churn-prediction/demo/demo.py [hunk 3]
```diff
@@ -32,21 +36,26 @@ def predict(*val):
                 "shape": [1]
             }
         )
+    # use the requests library, post the request and get the response
     resp = requests.post(
         url=urljoin(MODEL_DEPLOYED_URL, "v2/models/churn-model/infer"),
         json=json_body
     )
+    # convert the response into dictionary
```
Comment (nikp1172) [inline end-to-end-examples/churn-prediction/demo/demo.py:44]: remove

File: end-to-end-examples/churn-prediction/demo/demo.py [hunk 4]
```diff
@@ -32,21 +36,26 @@ def predict(*val):
...
     r = resp.json()
+    # return the output and model_version
     return [ r["outputs"][0]["data"][0],  r["model_version"]]
 
-
+# create description for the gradio application
```
Comment (nikp1172) [inline end-to-end-examples/churn-prediction/demo/demo.py:49]:
```suggestion
# description for the gradio application
```

File: end-to-end-examples/red-wine-classification/train/train.py [hunk 5]
```diff
@@ -56,4 +63,6 @@
     custom_metrics=[{"name": "log_loss", "type": "metric", "value_type": "float"}],
 )
 
-print(f"Logged model: {model_version.fqn}")
+e = time.time()
+print(f"################### Time taken to log models {e-s} ")
```
Comment (nikp1172) [inline end-to-end-examples/red-wine-classification/train/train.py:67]: remove all the "######"

File: end-to-end-examples/red-wine-classification/train/train.py [hunk 6]
```diff
@@ -7,45 +7,52 @@
...
 schema = {"features": features, "prediction": "categorical"}
 print(f"Schema: {schema}")
 
+# get the training and test data predictions
 y_pred_train = model.predict(X_train)
 y_pred_test = model.predict(X_test)
 
 # logging the data for experiment tracking
-# You can push the model to your choice of storage or model registry.
+# you can push the model to your choice of storage or model registry.
```
Comment (nikp1172) [inline end-to-end-examples/red-wine-classification/train/train.py:26]: remove

**Pull request merged**